### PR TITLE
Implementing 040720.

### PR DESCRIPTION
**Pull request by mikerenfro**
Implementing RHEL-07-040720.
Modifying RHEL-07-040700 to install tftp and tftp server when required.

### CODE REVIEW

File: tasks/fix-cat1.yml [hunk 1]
```diff
@@ -408,9 +408,8 @@
       name:
           - tftp
           - tftp-server
-      state: absent
+      state: "{{rhel7stig_tftp_required | ternary('present', 'absent') }}"
```
Comment (jamescassell) [inline tasks/fix-cat1.yml]: I don't think we should install anything here.

File: tasks/fix-cat2.yml [hunk 2]
```diff
@@ -1952,9 +1952,15 @@
...
 - name: "MEDIUM | RHEL-07-040720 | PATCH | If the Trivial File Transfer Protocol (TFTP) server is required, the TFTP daemon must be configured to operate in secure mode."
-  command: "true"
-  changed_when: no
-  when: rhel_07_040720
+  lineinfile:
+    path: /etc/xinetd.d/tftp
+    regexp: "(?i)^.*server_args.*="
+    line: "\tserver_args\t\t= -s /var/lib/tftpboot"
+    insertafter: "\tserver\t\t\t="
+    state: present
```
Comment (jamescassell) [inline tasks/fix-cat2.yml]: then I'd add:
```
register: result
failed_when:
    - result | failed
    - result.rc != 257
```
so you can just ignore the case where the file is not present

File: tasks/fix-cat1.yml [hunk 3]
```diff
@@ -408,9 +408,8 @@
...
   when:
-      - not rhel7stig_tftp_required
```
Comment (jamescassell) [inline tasks/fix-cat1.yml]: I'd leave this.

File: tasks/fix-cat1.yml (hunk 1 above)
Comment (shepdelacreme) [reply tasks/fix-cat1.yml]: I concur I don’t think the role should install these packages when the “required” variable is set to true. Should just remove when required = false. The package/service install is up to another role.

File: tasks/fix-cat1.yml [hunk 4]
```diff
@@ -410,8 +410,8 @@
           - tftp-server
       state: absent
   when:
-      - not rhel7stig_tftp_required
       - rhel_07_040700
+      - not rhel7stig_tftp_required
```
Comment (jamescassell) [inline tasks/fix-cat1.yml]: I'd prefer not to relocate this line...

**Pull request merged**